### ISSUE TITLE
Can't deploy 1.8.0 directory not found

### ISSUE THREAD

**Issue by Demaz93** [DefectDojo/django-DefectDojo#2945]
**Bug description**
During deploy in Kubernetes I receive an error from django pod caused by an error in initialization script in version 1.8.0.

Here: https://github.com/DefectDojo/django-DefectDojo/blob/master/docker/entrypoint-nginx.sh#L35
and Here: https://github.com/DefectDojo/django-DefectDojo/blob/master/docker/entrypoint-nginx.sh#L36

is indicated this path: /run/defectdojo
but the folder defectdojo doesn't exist so is impossible to deploy the pod

**Deployment method** *(select with an `X`)*
- [X] Kubernetes
- [ ] Docker
- [ ] setup.bash / legacy-setup.bash

**Comment by valentijnscholten** [comment]
@madchap @dsever

**Comment by dsever** [comment]
@valentijnscholten I have seen it before, but as far i remember it has been fixed by @madchap in meanwhile.

**Comment by madchap** [comment]
Yes, this is fixed in `dev`, so if you're using a release tag then you should be good in just a few days.
